CONTRIBUTING.md

```diff
@@ -89,8 +87,12 @@ We use and recommend the following workflow:
 6. Run the relevant scripts in [the section below](https://github.com/microsoft/promptflow/blob/main/CONTRIBUTING.md#dev-scripts) to ensure that your build is clean and all tests are passing.
 7. Create a PR against the repository's **main** branch.
    - State in the description what issue or improvement your change is addressing.
+   - Link the PR to the issue in step 1.
    - Verify that all the Continuous Integration checks are passing.
 8. Wait for feedback or approval of your changes from the code maintainers.
+   - If there is no response for a few days, you can create an issue to raise awareness.
```

Comment:
We already mentioned create an issue at line 75, then what's the create an issue here? same one or a new one?

Reply:
A new one issue here.

For example, a contributor created an issue, discussed with us and assigned to himself, then we might lose our attention to this issue (and its related PR); for this case, he can raise the ad-hoc issue for our attention.

Reply:
Also, we explain our daily triage process here, so contributors can somehow know how to raise their awareness, no matter what they want to catch our eyes.

Reply:
offline synced, updated to highlight the new issue